shared/pingpong/accounts.go

```diff
@@ -461,7 +461,67 @@ func genBigNoOpAndBigHashes(numOps uint32, numHashes uint32, hashSize string) []
 	return ops.Program
 }
 
-func genAppProgram(numOps uint32, numHashes uint32, hashSize string, numGlobalKeys uint32, numLocalKeys uint32) ([]byte, string) {
+func genAppProgram(numOps uint32, numHashes uint32, hashSize string, numGlobalKeys, numLocalKeys, numBoxUpdate, numBoxRead uint32) ([]byte, string) {
+	if numBoxUpdate != 0 || numBoxRead != 0 {
+		prologue := `#pragma version 8
```

Comment:
@algoidurovic Echoing a verbal we had:  Somewhere in the PR, can you add prose (either inline comments or a README.md) explaining behavior isolated by update vs read workloads?

The goal is to help pingpong operators build an intuition for how reads vs updates impacts algod performance + to help triage performance regressions.

Reply:
Added a comment to `pingpong/runCmd.go` as well as a README to hopefully make things easier for new users.

Reply:
@algoidurovic Resolving - thanks for the added commentary.